docs/jwst.rst

```diff
@@ -0,0 +1,254 @@
+.. _jwst_instruments:
+
+
+*****************************
+JWST Instrument Model Details
+*****************************
+
+
+The following describes specific details for the various JWST instrument classes. See also :ref:`the references page <references>` for information on data sources. 
+
+
+One general note is that the webbpsf class interfaces do not attempt to exactly
```

Comment:
Fair point. I was mentally thinking "package name" here when talking about class interfaces. But consistency is good.